docs/design.md

```diff
@@ -43,8 +43,9 @@ The dialect determines what SQL is generated for various Tableau actions.
 
 Choosing the right dialect is a critical part of writing a connector. For example:
 
-- If you set <span style="font-family: courier new">base</span> to "mysql_odbc" as the parent, you can skip configuring a dialect, and your connector will use the parent’s dialect.
-- If your database follows the SQL standards of another database that Tableau already supports (listed below), then you can choose that dialect as a starting point.
+- Use superclass's dialect scenario: If you set <span style="font-family: courier new">base</span> to "mysql_odbc" as the parent, you can skip configuring a dialect, and your connector will use the parent’s dialect.
+- Inherit a base dialect scenario: If your database follows the SQL standards of another database that Tableau already supports (listed below), then you can choose that dialect as a starting point, you can also add your own customizations beyond the base dialect.
```

Comment:
I'd add a warning, either here or in the "Should I create a dialect definition" section below (or preferably both), that a new version of Tableau may change the base dialects and cause unintended side effects for connectors inheriting from the base dialect. We want to discourage people from tying their connector to our connectors as much as possible.

Reply:
Done, thank you!